### ISSUE TITLE
add "any" or multiple choice options for photo license selector on obs search

### ISSUE THREAD

**Issue by kueda** [inaturalist/inaturalist#873]
I don't think anyone is ever really searching for a single license. Usually you just want "something I can use" or "something I can use without worrying about those damn no-derivs and share-alike clauses."

**Comment by pleary** [comment]
How would this be presented to the user? I could see having multiple things the user could toggle like 'no rights reserved', 'NC', 'ND', 'SA', and each option could be yes or no. For example:
 - show me all obs with NC licenses
 - show me all obs w/o ND licenses

But that could be a complicated UI. I'm less concerned about the UI right now, rather the functionality it will expose that we need to implement in the API.

**Comment by kueda** [comment]
Yeah, I'm really just interested in a multiple choice UI, so checkboxes or what we have for the months chooser. From a UI perspective it would be nice to have a pure angular solution instead of the jQuery multiselect. From the API perspective, though, I think we just need to be able to specify multiple licenses.

Also, this would just be for photo licenses, not obs.

**Comment by pleary** [comment]
ok got it. The API can handle this already with `/observations?photo_license=cc-by,cc-by-nd` so looks like we'll just need changes on the Rails end to list the licenses the user can choose from and make sure they're sent properly to the API
